File: frontend/Models/TNGUOIDUNGTHEOCUAHANG.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class TNGUOIDUNGTHEOCUAHANG
    {
        public int? Id { get; set; }
        public string Note { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? UsercreatedId { get; set; }
        public int? SuserId { get; set; }
        public int? DcuahangId { get; set; }
    }
}

File: backend/src/models/TNGUOIDUNGTHEOCUAHANG.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class TNGUOIDUNGTHEOCUAHANG extends Model {
    public id!: any;
    public note!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public usercreatedid!: any;
    public suserid!: any;
    public dcuahangid!: any;
}

TNGUOIDUNGTHEOCUAHANG.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    SUSERID: {
        type: DataTypes.INTEGER,
        
    },
    DCUAHANGID: {
        type: DataTypes.INTEGER,
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'TNGUOIDUNGTHEOCUAHANG',
    tableName: 'TNGUOIDUNGTHEOCUAHANG',
    timestamps: false
});

export default TNGUOIDUNGTHEOCUAHANG;
